float32 fear
float32 surprise
float32 interest
float32 relief